components/CropSelector/CropSelector.vue

```diff
@@ -13,6 +13,7 @@
       v-on:valid="handleValid($event)"
       v-on:add-clicked="handleAddClicked"
       v-bind:includeAddButton="canCreateCrop"
+      v-bind:popupUrl="popupUrl"
```

Comment:
See comment in `SelectorBase` about `popupUrl` and `includeAddButton`.  I think they are redundant.  Here you could use a conditional or a computed property to include the `popupUrl` if `canCreateCrop` is `true` and `null` if it is `false`.

This would apply to all other parent components also.